graphql/util.go

```diff
@@ -408,6 +408,14 @@ func SchedulePatch(ctx context.Context, patchId string, version *model.Version,
 			return errors.Wrap(err, "Error setting patch variants and tasks"), http.StatusInternalServerError, "", ""
 		}
 
+		projectRef, err := model.FindOneProjectRef(project.Identifier)
+		if err != nil {
+			return errors.Wrap(err, "unable to find project ref"), http.StatusInternalServerError, "", ""
+		}
+		if err := units.ProcessTriggerAliases(ctx, p, projectRef, evergreen.GetEnvironment(), patchTriggerAliases); err != nil {
```

Comment:
I'm not sure it's a great idea to call this here. ProcessTriggerAliases creates and runs jobs right away, which I don't think should be done as part of graphql. Should ProcessTriggerAliases itself be a job, or do we need this to run before FinalizePatch (given some of the comments in that function)? In which case I think we should add some comments here to make it clear why we need to do this in the moment. cc @malikchaya2

Reply:
This can't be done in finishPatch where it was being done until now (for the CLI). However, I think we may be able to move this into the if p.IsParent() {} block in finalize patch.

Reply:
Yes definitely worth pointing this out and I'll add some commentst! For context, Chaya and I met with Jonathan and determined that it pretty much is necessary to create and run child patch jobs in one sweep. When downstream tasks are created with `--triggerAlias` with the CLI, the intent can be created immediately and then the job can be run later. This code supports a different flow, though, where a patch has been created without the flag and a user would like to add downstream tasks using the UI. In that case, I think it's pretty much necessary to immediately create the job and run it. We acknowledged that this may result in some slowness, but the CLI remains available for a speedier option.

Let me know if this makes sense or if I misunderstood! Also if you have any other thoughts about relocating this I'm all ears.

Reply:
Oh sorry, just remembered my thinking here -- I didn't call `ProcessTriggerAliases` in `FinalizePatch` because `finishPatch` (which is invoked by the CLI) calls `FinalizePatch`. This seemed to me like the easiest way to avoid a duplicate operation.

Reply:
It might be worth duplicating anyway; can a user change the aliases they passed in from the CLI?
I'm fine with not moving it into FinalizePatch if that's too messy; I'd like to still add comments here though in that case.

Reply:
No, I believe we decided that for the current scope the UI will only allow adding new downstream tasks. Modifying CLI-specified options could be added later if we hear from users. I think I'll keep the location for now, comments incoming 👍 thanks!